JSON/Integer Trees.json

```diff
@@ -1,7 +1,6 @@
 {
   "Compatibility": "Compatible",
   "Contributors": {
-    "Developer": [ "Deaf" ],
```

Comment:
I feel like this may break things, considering that repo only shows Manual writers and Developers. It's likely that the author field will be shown as empty.